### PR TITLE
fix #693 + some bonus stuff

### LINKED ISSUE #693: Is this too much to add 3 more software (and their icons) ?
## Bug or Feature Request

I use 2 of 3 programs that listed below for blocking ads and tracking hosts via hosts file

#### hostsmgr
- Platform: Windows (since XP SP3 and above)
- Syntax support: Hosts File (localhost), Domain Name
- License: GPL v3
- Language: C/C++
- WebSite: https://www.henrypp.org/product/hostsmgr
- Source Code: https://github.com/henrypp/hostsmgr
  > Icon image inside the executable
  > image

#### personalDNSFilter
- Platform: Windows (JRE) / Android (4.0 and above)
- Syntax support: Hosts File (localhost), Domain Name
- License: GPL v2
- Language: Java
- Website: https://zenz-solutions.de/personaldnsfilter
- Source Code: https://github.com/IngoZenz/personaldnsfilter 
  > You can see icon image from website

#### Blokada
- Platform: Android (5.0 and above)
- Syntax support: Hosts File (localhost)
  > Not sure it's also support domain name or not
- License: GPL v3
- Language: Kotlin / Java
- Website: https://blokada.org
- Source Code: https://github.com/blokadaorg/blokada 
  > You can see icon image from website

### PR DESCRIPTION
**Pull request by DandelionSprout**
I've now added the softwares, the syntaxes they support, and their images. Note that I could not find anything about *hostsmgr* on its websites that indicated that it actively processed DNS addresses, let alone multiple ones.

As added bonuses, I've also:
* Added "DNS addresses" as syntax №22.
* I was very close to separating `[Adblock Plus 2.0]` and `[Adblock Plus 1.1]` into separate syntaxes, until I realised that no sane adblocker tool worth its salt did not support the 2.0 syntax in the year 2019. I've therefore instead *combined* them into a versionless syntax for the time being.
* * I am aware that `[Adblock Plus 3.1]` is a thing now, and which uBO refuses to have full support for, although the main version of *ABP Anti-Circumvention List* is the only list in the known world to currently use it, and even then does uBO support ~60% of its entries.

### CODE REVIEW

**Comment by DandelionSprout** [comment]
Ping #693, as mentioning that issue in this pull's title was not enough for it to be linked to.

File: src/FilterLists.Web/ClientApp/modules/home/components/softwareIcon/SoftwareIcon.tsx [hunk 1]
```diff
@@ -67,5 +70,8 @@ const icons: { [id: number]: IIcon; } = {
     22: { image: img22, imageTitle: "Diversion" },
     23: { image: img23, imageTitle: "dnsmasq" },
     24: { image: img24, imageTitle: "Slimjet" },
-    25: { image: img25, imageTitle: "uMatrix" }
+    25: { image: img25, imageTitle: "uMatrix" },
+    26: { image: img24, imageTitle: "Blokada" },
```
Comment (collinbarrett) [inline src/FilterLists.Web/ClientApp/modules/home/components/softwareIcon/SoftwareIcon.tsx]: `img24` seems to be repeated several times. Probably just a copy/paste error.

Comment (DandelionSprout) [reply src/FilterLists.Web/ClientApp/modules/home/components/softwareIcon/SoftwareIcon.tsx]: Ah. I had forgot to change the numbers after copying and pasting an existing line. Well caught. Should be fixed now.

**Comment by kowith337** [comment]
FYI: `hostsmgr` is  windows app and just generate hosts file and replace the system hosts file (where it's locate at `C:\Windows\system32\drivers\etc\hosts`) or export to custom location, none of anything that related to DNS.

**Comment by DandelionSprout** [comment]
Ah, okay. I see now that I was *somehow* able to misinterpret "Syntax support: (...) Domain Name" in https://github.com/collinbarrett/FilterLists/issues/693#issue-410425858 as referring to DNS addresses, which is a pretty big oversight of mine by my standards.

**Pull request merged**